src/Aspire.Dashboard/Utils/FormatHelpers.cs

```diff
@@ -26,22 +27,48 @@ static string GetLongTimePatternWithMilliseconds()
     private static partial Regex MatchSecondsInTimeFormatPattern();
 
     private static readonly string s_longTimePatternWithMilliseconds = GetLongTimePatternWithMilliseconds();
+    private static readonly string s_shortDateLongTimePatternWithMilliseconds = DateTimeFormatInfo.CurrentInfo.ShortDatePattern + " " + GetLongTimePatternWithMilliseconds();
```

Comment:
Can we reuse the previously computed value to avoid running the regex twice?

```suggestion
    private static readonly string s_shortDateLongTimePatternWithMilliseconds = DateTimeFormatInfo.CurrentInfo.ShortDatePattern + " " + s_longTimePatternWithMilliseconds;
```